products/fuse/get-started.adoc

```diff
@@ -6,7 +6,6 @@ Prerequisites &#38; Required Dependencies Checklist
 
 == Prerequisites section
 1. Java SE Development Kit (JDK), we recommend OpenJDK or Oracle JDK.
-2. JBoss&#174; Developer Studio 9.0
```

Comment:
Sorry, I was wrong. You need to remove all the pre-reqs. Notice that the JDK install is mentioned here: https://github.com/redhat-developer/developers.redhat.com/pull/736/files#diff-4969c2a46f168bb48a422e3abe0cf507R21